### PR TITLE
Netman testing

### PR DESCRIPTION
**Pull request by crizzitello**
This Pr does two things 
 1. Create new methods in NetMan to allow handling of connection test
 2. Combine `stopReply` and `tidyReply` into `cleanUpReply`

### CODE REVIEW

File: src/helpers/cleanupreply.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2020, Verizon Media
```
Comment (JoelAtDeluxe) [inline src/helpers/cleanupreply.h]: Copyright?

File: src/helpers/cleanupreply.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#pragma once
+
+#include <QNetworkReply>
+
+static void cleanUpReply(QNetworkReply **reply) {
+  if (*reply == nullptr) {
+    return;
+  }
+
+  (*reply)->isFinished() ? (*reply)->close() : (*reply)->abort();
```
Comment (JoelAtDeluxe) [inline src/helpers/cleanupreply.h:12]: Yeah, that's clearly better.

**Review by JoelAtDeluxe — COMMENTED** [review]
Looks good to me.

File: src/helpers/cleanupreply.h (hunk 1 above)
Comment (crizzitello) [reply src/helpers/cleanupreply.h]: It was in the file i moved should i remove it ?

Comment (JoelAtDeluxe) [reply src/helpers/cleanupreply.h]: That I don't know, but both the year and the company don't seem right anymore. I don't know how useful these copyrights are anyway. Maybe Joe or John has an opinion?

**Review by jkennedyvz — APPROVED** [review]

**Pull request merged**